tcms/rpc/__init__.py

```diff
@@ -0,0 +1 @@
+DATE_TIME_FORMATS = ['%Y-%m-%d', '%Y-%m-%d %H:%M:%S']
```

Comment:
Make this directly the value of `input_formats` in `forms/__init__.py`. Doesn't make sense to have a separate constant which isn't used anywhere else, moreover in another file.